.github/workflows/linux-build.yml

```
@@ -50,9 +49,122 @@ permissions:
+          cd velox/docs
+          make clean
+          make html
+
+      - name: CCache after
+        run: |
+          ccache -vs
+
+      - uses: assignUser/stash/save@v1
+        with:
+          path: '${{ env.CCACHE_DIR }}'
+          key: ccache-ubuntu-debug-default-${{ inputs.use-clang && 'clang' || 'gcc' }}
```

actionlint (1.7.7)

property "use-clang" is not defined in object type {}

(expression)